### ISSUE TITLE
Game names should be larger at forum topics and posts

### ISSUE THREAD

**Issue by elifkizilky** [bounswe/bounswe2023group7#548]
### Issue Description

As far as i remember, one of our suggestions was making the game names in the posts and topics make larger and more striking. We can enhance our css style accordingly. What do you think @yunusemrealtug @mtkamiloglu ?

### Deadline of the Issue

26.11.2023 21.59

### Deadline for the Review

26.11.2023 23.59

**Comment by yunusemrealtug** [comment]
Yeah maybe this will enable our project to be perceived as domain spesific, but in the forum, I thought the user's feedback about a game is more important than the game's name. Therefore, I put game's name with smaller textsize. However, this is a design choice, if our customers think that the game should be at the forefront more, it should be.

**Comment by elifkizilky** [comment]
I actually did it. In the forums page under the trending and latest topic, I added a background color:
![image](https://github.com/bounswe/bounswe2023group7/assets/69367633/918debee-4ede-460b-a143-193b54bf1748)


In the posts aka threads, I completely change the design and structure so actually this issue is kind of resolved. I will add its link to PR when it is finished.

**Comment by elifkizilky** [comment]
I did even better 😄  Here we are displaying the user avatar too, and also enhanced the other little components. It is a better design, i think
![image](https://github.com/bounswe/bounswe2023group7/assets/69367633/fe6e647b-c107-4aa6-92eb-9542cd907a71)
